### ISSUE TITLE
Way to position messages

### ISSUE THREAD

**Issue by zeelot** [zeelot/kohana-yform#22]
mostly, there seems to be a constant need to place a message before a label, after a label (before input), and after the input... should this always require me to define a different view? should I just define 3 views for the 3 different positions? or just pass another variable into the view like $position (like this the most)

**Comment by dtb** [comment]
I think a variable makes sense, so that you don't have to keep a bunch of views in sync.

It would be really nice if this was configurable on a per-message-type basis.

**Comment by zeelot** [comment]
yeah I thought about that need right after making the issue. trying not to add too much bloat but I'll see what I can do. I might bring back the messages class in order to have more options than with a regular array
